packages/video_player/video_player_avfoundation/ios/Classes/Cache/Cache/CacheAction.h

```diff
@@ -0,0 +1,16 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Foundation/Foundation.h>
+
+typedef NS_ENUM(NSInteger, CacheType) { CacheTypeLocal = 0, CacheTypeRemote };
```

Comment:
what does local vs remote mean? can you comment here?

Reply:
Changed the types